be2-scala/src/main/scala/ch/epfl/pop/pubsub/PubSubMediator.scala

```diff
@@ -14,6 +14,7 @@ class PubSubMediator extends Actor with ActorLogging {
   private var channelMap: mutable.Map[Channel, mutable.Set[ActorRef]] = mutable.Map.empty
 
   override def receive: Receive = LoggingReceive {
+    // FIXME pubsub mediator doesnt check that the channel exists in db
```

Comment:
I'd fix this before merging

Reply:
Is this resolved in the other PR too? 
If so, feel free to merge.

Reply:
Resolved #449